codebase/composer.json

```diff
@@ -51,11 +51,13 @@
         "drupal/group": "^3.2",
         "drupal/hal": "^1.0||^2.0",
         "drupal/matomo": "^1.19",
+        "drupal/metatag": "^1.26",
         "drupal/openid_connect": "^1.2",
         "drupal/pdf": "^1.1",
         "drupal/pf_email": "^2.1",
         "drupal/push_framework": "^2.2",
         "drupal/rest_oai_pmh": "^2.0@beta",
+        "drupal/robotstxt": "^1.5",
```

Comment:
please can you advise what the metatag and robotstxt modules are used for?

Reply:
metatag is used for allowing the admins to put **noindex** tag on all the pages, as well as to apply custom tags.
robotstxt module is not used at the moment, but it will help the admin's to change the current robots.txt via admin panel.